app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -460,16 +460,16 @@ protected void setMediaPreviews(final List<Attachment> attachments, boolean sens
 
         mediaPreview.setAspectRatios(AttachmentHelper.aspectRatios(attachments));
 
-        mediaPreview.forEachIndexed((i, imageView) -> {
+        mediaPreview.forEachIndexed((i, imageView, descriptionIndicator) -> {
             Attachment attachment = attachments.get(i);
             String previewUrl = attachment.getPreviewUrl();
             String description = attachment.getDescription();
 
             if (TextUtils.isEmpty(description)) {
-                imageView.setContentDescription(imageView.getContext()
-                        .getString(R.string.action_view_media));
+                imageView.setContentDescription(imageView.getContext().getString(R.string.action_view_media));
```

Comment:
Views get recycled, `descriptionIndicator` needs to be hidden here or after a while of scrolling they will just be shown everywhere

Reply:
Done